zookeeper.spec

```diff
@@ -83,7 +83,7 @@ rm -rf %{buildroot}
 
 %files
 %defattr(-,root,root,-)
-%doc LICENSE.txt NOTICE.txt README.md
+%doc LICENSE.txt NOTICE.txt
```

Comment:
This file is still in the upstream and is the in release tarball for this version as well - https://github.com/apache/zookeeper/blob/master/README.md.